<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView_gameView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>911</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_informationsView">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_optionsView">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>180</y>
     <width>241</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_buttonView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>911</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>151</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelPlayer">
        <property name="text">
         <string>Player: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPlayerCurrent">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelPhase">
        <property name="text">
         <string>Phase:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPhaseCurrent">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
